app/views/shared/_alert.html.erb

```diff
@@ -1,5 +1 @@
-<% flash.each do |type, msg| %>
-  <div>
-    <%= msg %>
-  </div>
-<% end %>
+
```

Comment:
this file is now blank and wont render anything  should look something like this

 <%flash.each do |type, msg| %>
  <div>
    <%= msg %>
  </div>
<% end %>